libs/providers/ofrep-web/src/lib/ofrep-web-provider.spec.ts

```diff
@@ -0,0 +1,300 @@
+import { OfrepWebProvider } from './ofrep-web-provider';
+import TestLogger from '../../test/test-logger';
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import { server } from '../../../../shared/ofrep-core/src/test/mock-service-worker';
+import { ClientProviderEvents, ClientProviderStatus, OpenFeature } from '@openfeature/web-sdk';
+
+describe('OFREPWebProvider', () => {
+  beforeAll(() => server.listen());
+  afterEach(async () => {
+    server.resetHandlers();
+    await OpenFeature.close();
+  });
+  afterAll(() => server.close());
```

Comment:
This should use MSW client.
It is called worker I think: https://mswjs.io/docs/integrations/browser#setup

Reply:
No jest is acting only as a server and I've read somewhere that we should use the server with jest.

Will try to find the link tomorrow

Reply:
Mh I would have assumed to use JestDomEnv here

Reply:
Just had a look I understand what you are saying @beeme1mr. So you are right :)